## R0124 (comparison-with-itself)

### :x: Problematic code:

```python
value = 5
if value == value:
    pass
```

### :heavy_check_mark: Correct code:

```python
value = 5
if value:
    pass
```

### Rationale:

Used when something is compared against itself.

### Related resources:

- [Testcases](https://github.com/PyCQA/pylint/blob/master/tests/functional/l/logical_tautology.py)
- [Issue Tracker](https://github.com/PyCQA/pylint/issues?q=is%3Aissue+%22comparison-with-itself%22+OR+%22R0124%22)
